### PR TITLE
[Feature] Feature/title/luyoes title and more

### PR DESCRIPTION
**Pull request by ghost**
## Description

Add title, subTitle, titleStyle, subTitleStyle

## Related Issues

#22 

## Tests

I added the following tests:

X

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Card/index.tsx [hunk 1]
```diff
@@ -21,20 +21,63 @@ const StlyedImage = styled.Image`
...
   return (
     <Container style={[containerStyle]}>
       {image && <StlyedImage source={image} style={[imageStyle]} />}
+
+      <TitleContainer style={[titleContainerStyle]} titleTextVertical={ titleVertical } >
+        <TitleText style={[titleStyle]} > {title} </TitleText>
+        { subTitle && subTitle.length > 0
+          ? <SubTitleText style={[subTitleStyle]}> {subTitle} </SubTitleText>
+          : null }
+      </TitleContainer>
```
Comment (sk97kang) [inline main/Card/index.tsx]: I think it's better to mark only when there is.

```jsx
      {title && subtitle && <TitleContainer style={[titleContainerStyle]} titleTextVertical={ titleVertical } >
        {title && <TitleText style={[titleStyle]} > {title} </TitleText>}
        { subTitle && <SubTitleText style={[subTitleStyle]}> {subTitle} </SubTitleText>}
      </TitleContainer>}
```

File: main/Card/index.tsx [hunk 2]
```diff
@@ -44,6 +87,27 @@ const Card: FC<Props> = (props) => {
...
+    color: '#000000',
+
+  },
+  subTitleStyle: {
+    backgroundColor: 'transparent',
+    color: '#e4e4e4',
+    height: 20,
+  },
+
+};
```
Comment (sk97kang) [inline main/Card/index.tsx]: The default style would be better to be in the styled component.

File: main/Card/index.tsx (hunk 1 above)
Comment (LUYOES) [reply main/Card/index.tsx:79]: @sk97kang 
```
  {title && subtitle && <TitleContainer style={[titleContainerStyle]} ...
```

I think we'd better use 'or' for containers.

Comment (LUYOES) [reply main/Card/index.tsx:79]: const renderTitle = (title !== undefined && title.length > 0) || (subTitle !== undefined && subTitle.length > 0);

Comment (ghost) [reply main/Card/index.tsx]: I think we should use ||

**Pull request merged**